sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md

```diff
@@ -0,0 +1,73 @@
+# Migrating recordings to the `azure-sdk-assets` repository
+
+## Some background
+
+Recordings take up a large amount of space in our repository and generate a lot of churn. The asset sync project, owned by our Engineering System team, is a solution to this problem. Leveraging and extending the existing test proxy, the asset sync extension adds support for moving the recordings outside of the main azure-sdk-for-js repo.
+
+## Performing the migration
+
+The package you are migrating needs to be using the new version of the recorder that uses the test proxy (`@azure-tools/test-recorder@^2.0.0`). Most packages are using the new recorder already; if yours is not you should migrate as soon as possible. More detail on migrating to the new recorder can be found in the [recorder 2.0 migration guide].
+
+To migrate, there is just one command to run. With [`dev-tool`] and [Powershell] installed, run the following in your package directory:
```

Comment:
Update dev-tool with following? To make it simpler.

Navigate to `azure-sdk-for-js/common/tools/dev-tool`.
```
cd common/tools/dev-tool
npm install -g
```